lib/pluck_each.rb

```diff
@@ -16,10 +16,13 @@ def pluck_each(*args)
 
     def pluck_in_batches(*column_names)
       options = column_names.extract_options!
+      string_column_names = column_names.map(&:to_s)
 
       # Ensure the primary key is selected so we can use it as an offset
       # `pluck` already handles duplicate column names, and it keeps the first occurence
```

Comment:
This line no longer make sense since this checks if the primary key is included.